templates/docs/building-vanilla.md

```diff
@@ -10,8 +10,67 @@ context:
 
 Here you will find information on how you can use different tools to build Vanilla into production CSS.
 
+### dart-sass
+
+To get set up with [`dart-sass`](https://github.com/sass/dart-sass), add the `sass` and `vanilla-framework` packages to your project dependencies:
+
+```
+yarn add sass vanilla-framework
+```
+
+In the script that builds the CSS in your `package.json`, you should include the path to `node_modules` when looking for `@import`s. In this example, we have called the build script `"build-css"`:
+
+```
+"build-css": "sass -w --load-path=node_modules src:dist --style=compressed"
+```
+
+Make a folder `src/`, create a file inside called `style.scss` and import Vanilla:
+
+```
+@import vanilla-framework/scss/vanilla;
```

Comment:
I think this should be 
```suggestion
@import "vanilla-framework/scss/vanilla";
```

Reply:
:scream: yes - you're right, nice spot! updated